tests/Traits/ApiTestingTrait.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2021 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Tests\Traits;
+
+use Lcobucci\JWT\Parser;
+use OAT\SimpleRoster\Entity\User;
+use OAT\SimpleRoster\Tests\Utils\AuthenticationResponse;
```

Comment:
Suggestion: I'd rename the namespace of this class to `OAT\SimpleRoster\Tests\Helpers\Authentication` instead. I'd use helpers because that's the name used in most of the testing frameworks. Authentication because I'm afraid that we start to create classes here just because it's not a test class becoming into a not organized namespace. We could also reorganize the classes into the Trait namespace to this namespace, but I think that can be too much :(

Reply:
Agree, fixed. Let's discuss what to do with Traits later, but I agree